### ISSUE TITLE
NTR: Cholesterol Granuloma

### ISSUE THREAD

**Issue by callahantiff** [obophenotype/human-phenotype-ontology#4729]
**Preferred term label:**
NTR: Cholesterol Granuloma

**Synonyms**
Middle ear cholesterol granuloma, petrous apex cholesterol granuloma, cholesterin granuloma, cholesterin granuloma of middle ear, cholesterinic granuloma

**Definition (free text, please give PubMed ID)**
A benign tumor or cyst formed at the tip of the petrous apex, which is part of the temporal bone and is located next to the middle ear [PMID:17823547](https://www.ncbi.nlm.nih.gov/pubmed/17823547).

**Parent term (use hpo.jax.org/app)**
HP:0032252 Granuloma

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
ORCID: [0000-0002-8169-9049](https://orcid.org/0000-0002-8169-9049)

**Comment by pnrobinson** [comment]
This is a good term. I think we should also give it a parent somewhere in the ear hierarchy. The anatomical structure is this
https://pubs.rsna.org/doi/full/10.1148/rg.321105758
but it is mixed between different parts, making it difficult to model exactly. Let's take a closer look.

**Comment by pnrobinson** [comment]
Adding as HP_0032570

**Comment by callahantiff** [comment]
@pnrobinson Sorry I did not respond to this sooner. Thank you so much for adding the new term!!
